scripts/west_commands/runners/bossac.py

```diff
@@ -9,14 +9,17 @@
 import pickle
 import platform
 import subprocess
-import sys
 
 from runners.core import ZephyrBinaryRunner, RunnerCaps, BuildConfiguration
-from zephyr_ext_common import ZEPHYR_SCRIPTS
 
 # This is needed to load edt.pickle files.
-sys.path.append(str(ZEPHYR_SCRIPTS / 'dts'))
-import edtlib  # pylint: disable=unused-import
+try:
+    import edtlib  # pylint: disable=unused-import
+except ImportError:
+    # This can happen when building the documentation for the
+    # runners package if edtlib is not on sys.path. This is fine
+    # to ignore in that case.
+    pass
```

Comment:
A low-level log statement maybe in case this ever happens outside this case because of some bug?

Reply:
> A low-level log statement maybe in case this ever happens outside this case because of some bug?

I think this is an all or nothing thing; either we don't handle it at all or we handle it like runners are supposed to. I'll do the latter.

Reply:
> A low-level log statement maybe in case this ever happens outside this case because of some bug?

Mmmh, I'm going to do it your way actually, to avoid having to touch the test cases.